File: testdata/bench/swift/BenchmarkTest00006.swift
import Vapor
import SQLite3

func handler006(_ req: Request) throws -> String {
    let param = try req.parameters.require("search")
    var db: OpaquePointer?
    var stmt: OpaquePointer?
    sqlite3_open(":memory:", &db)
    let sql = "SELECT name FROM products WHERE description LIKE '%\(param)%'"
    sqlite3_prepare_v2(db, sql, -1, &stmt, nil)
    return "ok"
}

File: testdata/bench/cpp/BenchmarkTest00006.cpp
#include <cstdio>
#include <sqlite3.h>

void lookup(const char *param) {
    char query[512];
    snprintf(query, sizeof(query), "SELECT id, name FROM items WHERE category = '%s'", param);
    sqlite3 *db;
    sqlite3_open("app.db", &db);
    sqlite3_exec(db, query, nullptr, nullptr, nullptr);
    sqlite3_close(db);
}
